java/arcs/android/devtools/RawDevToolsMessage.kt

```diff
@@ -0,0 +1,14 @@
+package arcs.android.devtools
+
+import arcs.android.devtools.DevToolsMessage.Companion.RAW_MESSAGE
+
+/**
+ * An implementation of [DevToolsMessage] to pass raw [ProxyMessage]s.
+ */
+class RawDevToolsMessage(override val message: String) : DevToolsMessage {
+    override val kind: String = RAW_MESSAGE
+
+    override fun toJson(): String {
+        return "{kind: \"$kind\", message: \"$message\"}"
+    }
```

Comment:
This will quickly become unsustainable and is already likely error-prone. For example, what happens if `message` contains quotation marks? (it doesn't look like this attempts to escape them, and would result in invalid JSON). 

Please consider looking into JSON serialization tooling, rather than hand-rolling JSON messages.

Reply:
Moved to use Json which was already in third_party